lib/mumuki/domain/helpers/user.rb

```diff
@@ -49,9 +49,9 @@ def discusser_of?(organization)
   (Mumukit::Auth::Roles::ROLES - [:owner]).each do |role|
 
     # Assignes the #{role} role to this user
-    # for the given slug
-    define_method "make_#{role}_of!" do |slug|
-      add_permission! role, slug
+    # for the given `grant_like`
+    define_method "make_#{role}_of!" do |grant_like|
+      add_permission! role, grant_like.to_mumukit_grant
```

Comment:
This change is not actually necessary, but I added `to_mumukit_grant` to make the conversion more explicit